packages/lesswrong/lib/editor/make_editable.js

```diff
@@ -45,6 +46,17 @@ export const makeEditable = ({collection, options = {}}) => {
     formGroup,
     permissions,
     fieldName = "",
+    hintText = <div>
+      <div>
+        Write your {collection.options.singleResolverName} here. Select text for formatting options.
+      </div>
+      <div>
+        You can switch between rich text and markdown in your user settings)
+      </div>
+      <div>
+        You can switch between rich text and markdown in your user settings)
```

Comment:
Mismatched parenthesis in the message, and also part of the message appears twice in a row.